dashi/src/lib/api/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface ApiOptions {
```

Comment:
Can we move these types to the types folder?

Reply:
Ok!